beets/importer.py

```diff
@@ -530,6 +533,17 @@ def remove_duplicates(self, lib):
                 util.prune_dirs(os.path.dirname(item.path),
                                 lib.directory)
 
+    def set_fields(self):
+        set_fields_dict = config['import']['set_fields']
+        for field in set_fields_dict.keys():
+            value = set_fields_dict[field].get()
+            log.debug(u'Set field {1}={2} for {0}',
+                      displayable_path(self.paths),
+                      field,
+                      value)
+            self.album[field] = value
+            self.album.store()
```

Comment:
Scratch my previous comment in light of the ordering constraints, but I still recommend "un-indenting" this line so the album get stored only once to update all of its fields. (And the same below for items.)